### ISSUE TITLE
Edit screen allows changing of reference build (it should not)

### ISSUE THREAD

**Issue by MatthewMaher** [statgen/locuszoom-hosted#46]
On the main view page for a study, there is an 'Edit' button that allows the user/creator to edit the Label, Study Name, PMID, etc. 

It also allows the user to edit the build (GRCh37 vs GRCh38).  **This edit should not be allowed** since the reference build was used during initial upload/ingestion to assign RSIDs and genenames.  But no such reprocessing is done when changing the value on the Edit page.  This results in the Build label on the top of the Manhattan plot being inconsistent with the RSIDs and gene names shown below.

**Comment by abought** [comment]
Good catch. For whatever dev is reading this in the future: this is an edge case, and makes sense to fix. 

Someone *could* edit the field, then click "re-run ingest step".... but those actions are not automatically linked, which is confusing. 

Options include:
1. Making the file rerun ingest when specific metadata fields are changed (might require additional UI to tell user "this is a destructive action")
2. or: Don't make gwas build editable.


There had been vague plans to make more fields editable, such as plot options (in case someone selected the wrong columns and wanted to re-ingest without re-uploading). This got sidelined for various reasons (including "we added UI to preview parser before upload"), so option 2 is probably the most pragmatic/consistent.
